src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -313,7 +313,14 @@ protected virtual void Dispose(bool disposing)
                 ScriptHost[] instances = GetLiveInstancesAndClear();
                 foreach (var instance in instances)
                 {
-                    instance.Dispose();
+                    try
+                    {
+                        instance.Dispose();
+                    }
+                    catch (Exception exc) when (!exc.IsFatal())
+                    {
+                        // Best effort
+                    }
```

Comment:
Not related to this issue, but impacted the tests we had to run.
This is related to some recent changes we made with the host initialization,